### DISCUSSION TITLE
[Projects Beta]Searching issues by project is not supported

### DISCUSSION THREAD

**Discussion by ssst0n3** [community/community#11694] [Projects and Issues]
> You can use the project qualifier to find issues that are associated with a specific [project board](https://docs.github.com/en/articles/about-project-boards) in a repository or organization. You must search project boards by the project board number. You can find the project board number at the end of a project board's URL.
> 
> https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-project-board

I can search issues by project for "projects" but not for "projects beta"

**Comment by wolfs** [comment]
We are currently using labels (something like `@<team>`) for marking issues belonging to a specific team. Some of the teams now migrated to using projects (beta) for their boards. So it is somewhat redundant for those teams to add issues to their board and to add an `@<team>` label to the issue to find all their issues in a search. Adding search support for project (beta) in the issue search would help us here as well.
